File: core/core-api/src/main/java/com/coffee/gu/api/controller/v1/response/order/OrderLineResponse.java
package com.coffee.gu.api.controller.v1.response.order;

import java.math.BigDecimal;

public record OrderLineResponse(
        Long menuId,
        String menuName,
        String imageUrl,
        String description,
        Long quantity,
        BigDecimal unitPrice,
        BigDecimal totalPrice
) {
}

File: core/core-api/src/main/kotlin/com/coffee/gu/api/controller/v1/response/order/OrderLineResponse.kt
package com.coffee.gu.api.controller.v1.response.order

import java.math.BigDecimal

class OrderLineResponse(
    val menuId: Long,
    val menuName: String,
    val imageUrl: String,
    val description: String,
    val quantity: Long,
    val unitPrice: BigDecimal,
    val totalPrice: BigDecimal,
)
